Cargo.toml

```diff
@@ -42,3 +45,6 @@ serial_test = "1"
 name = "wadm"
 path = "bin/main.rs"
 required-features = ["cli"]
+
+[patch.crates-io]
+wasmcloud-control-interface = { git = "https://github.com/thomastaylor312/control-interface-client.git", branch = "chore/update_rpc_version" }
```

Comment:
I'll change this once https://github.com/wasmCloud/control-interface-client/pull/44 is merged